pegasus/sites.v3/code.org/public/css/ai-pl-webinar-styles.scss

```diff
@@ -0,0 +1,108 @@
+// Custom styles for code.org/ai/pl/webinar
+// — supplemental to shared/css/design-system-pegasus.scss
+
+@import 'color', 'font', 'typography', 'breakpoints', 'barlow-semi-condensed-font';
+
+.hero-ai-pl-webinar {
+  background-image: url(/images/ai/pl/webinar/hero--1.png), url(/images/ai/pl/webinar/hero--2.png);
+  background-position: left top, right bottom;
+  background-repeat: no-repeat, no-repeat;
+  background-color: #360161;
+  //background-size: contain;
+  background-size: 250px, 500px;
+  &__by {
+    display: flex;
+    align-items: center;
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    margin-left: 32px;
+    flex-wrap: wrap;
+
+    img {
+      margin: 16px 32px 16px 0;
+    }
+  }
+
+  .text-wrapper p {
+    font-size: 20px;
+    margin-bottom: 30px;
+  }
+
+  @media (max-width: 1499px) {
```

Comment:
There are [breakpoint variables here](https://github.com/code-dot-org/code-dot-org/blob/40781fa722e53b066b90b5fef5b6fd25b7bb0581/shared/css/breakpoints.scss) you can use instead of pixel values if you'd like.

And these are the most common screen resolutions for visitors to our site:

<img width="185" alt="Screenshot 2023-07-11 at 9 39 52 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/1ff9ad09-b322-4d20-8190-ffc7350be812">

Reply:
partly done